### PR TITLE
Fix: Refactored parsers

### PR DESCRIPTION
**Pull request by angelotadeucci**
- Renamed ExpTableParser to ExpParser;
- Refactored Insignia, Quest and Exp Parsers to no longer user XmlReader;
- Optimized ScriptParser to ignore events scripts from CN, JP and KR;
- Used dotnet-format;
- New hashes.

### CODE REVIEW

File: GameDataParser/Parsers/ExpParser.cs [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 namespace GameDataParser.Parsers
 {
-    class ExpTableParser : Exporter<List<ExpMetadata>>
+    class ExpParser : Exporter<List<ExpMetadata>>
```
Comment (Sparkymod) [inline GameDataParser/Parsers/ExpParser.cs:9]: Why is this changing again? From `ExpParser` to`ExpTableParser` then again `ExpParser`?

File: Maple2Storage/Types/Metadata/ScriptMetadata.cs [hunk 2]
```diff
@@ -127,29 +127,26 @@ public override string ToString()
     public class Content
     {
         [XmlElement(Order = 1)]
-        public long Script;
```
Comment (Sparkymod) [inline Maple2Storage/Types/Metadata/ScriptMetadata.cs:130]: Why is this removed? `Script` must have an Id.

Comment (angelotadeucci) [reply Maple2Storage/Types/Metadata/ScriptMetadata.cs:130]: You are right, it was a mistake.

Comment (angelotadeucci) [reply Maple2Storage/Types/Metadata/ScriptMetadata.cs:130]: The script in Content refers to a long that the server doesn't need to know about it.
Example: 
```
    <script randomPick="1" id="10" feature="" locale="">
		<content voiceID="ko/Npc/00000079" **text="$script:0831180407000003$"** />
	</script>
```
The important part is ID for the Element Script not the Content Script.

**Review by Sparkymod — APPROVED** [review]

**Review by AlanMorel — APPROVED** [review]
